jaeger-core/src/test/java/com/uber/jaeger/senders/UdpSenderTest.java

```diff
@@ -64,7 +68,9 @@ private TestTServer startServer() throws Exception {
     Thread t = new Thread(server);
     t.start();
 
-    Thread.sleep(5);
+    // wait up to 5000ms to get this thread started
+    await().with().pollDelay(5, TimeUnit.MILLISECONDS).pollInterval(5, TimeUnit.MILLISECONDS).atMost(
+            5000, TimeUnit.MILLISECONDS).until(() -> t.isAlive());
```

Comment:
does it matter if we use `pollDelay`? I'd prefer to keep it as simple as possible. Also in Go code we typically poll every 1ms, for up to 1sec (here you have 5sec, which is fine too).

nit: can you use `atMost(5, TimeUnit.SECOND)` instead of ms?